framework/test/mocha/integration/transactions/exceptions/duplicate_signatures.js

```diff
@@ -0,0 +1,272 @@
+const expect = require('chai').expect;
```

Comment:
```suggestion
const { expect } = require('chai');
```
Also, header required